browser/brave_tab_helpers.cc

```diff
@@ -149,6 +155,12 @@ void AttachTabHelpers(content::WebContents* web_contents) {
   }
 #endif
 
+#if BUILDFLAG(ENABLE_AI_REWRITER)
+  if (ai_rewriter::features::IsAIRewriterEnabled()) {
+    ai_rewriter::AIRewriterTabHelper::CreateForWebContents(web_contents);
```

Comment:
Q) Do we really need tab helper for this?
This tab helper is used for show/hide.
So I think it could be handled by instance per browser instead of having instance per tab.
Or it there a limitation for using mojo?

Reply:
Strictly speaking we don't need a tab helper for this, but we want to end up with one button per |web_contents|, not per-frame and this feels like quite a natural way of achieving this.